### ISSUE TITLE
[Question] Publicly available instance running?

### ISSUE THREAD

**Issue by TravisL12** [PawelPleskaczynski/apod_api#4]
Hey there, I saw a post about your Node API version of APOD on `http://asterisk.apod.com/`. I have a chrome extension that uses the NASA API but it's pretty slow. I was curious if you had an instance of your API hosted somewhere that I could use for my extension? Otherwise I could also try hosting it myself. Thanks!

**Comment by PawelPleskaczynski** [comment]
Hi, thank you for checking out my project! I have this API hosted on Heroku, here's the link to it: https://apodapi.herokuapp.com/. No API key is needed, you can use it freely. I'm not sure if apps hosted on free tier Heroku instances have some limits, but it seems that request number or bandwidth is not limited, so it should be alright. Also, I have observed occasional downtimes, but they also happen at exactly the same time on the official API, so that is just the APOD website that goes down for some reason. Anyways, if you have any questions about this, feel free to ask!

**Comment by TravisL12** [comment]
Sounds good, I'll make a build of my extension using your API and all of it's additional features (keyword search!). And let you know how it goes and if I decide to publish a release using it. My traffic isn't very high (~450 active users) though Heroku can easily handle that plus much more.

**Comment by PawelPleskaczynski** [comment]
Sure, I'm waiting for your feedback! Greetings from Poland by the way :smiley:

**Comment by TravisL12** [comment]
I've moved my extension over to your API and the nice thing is that almost nothing changed. Which is a good thing. I also integrated your search endpoint instead of my old way of scraping the results from the actual [APOD search](https://apod.nasa.gov/cgi-bin/apod/apod_search) (which is a terrible way to do this but I didn't have any other option). It seems like it might be faster but hard to say, to counterbalance the slow load speed of the NASA API I've utilized web workers to cache images on load and other tricks. So with more use I hope to recognize that your API is overall faster.

The only small thing is that your response for the APOD's uses `description` instead of `explanation` but this is very easily adjusted for. I was thinking that it'd be nice to primarily use your API but to have a fallback to the official API in case of any downtime, but that's more work than is necessary and I have the impression you're not removing your API anytime soon.

Anyway, I will likely move forward with a release using your API and any thoughts you have on the extension I'm happy to hear, I don't really have any grand feature ideas since it's a pretty basic concept.

APOD By The Trav (I'm thinking changing the name soon)
[Chrome](https://chrome.google.com/webstore/detail/astronomy-picture-of-the/aedpginojmhafbemcoelnppdcmlfjcdj)
[Firefox](https://addons.mozilla.org/en-US/firefox/addon/apod-by-the-trav/)

**Comment by PawelPleskaczynski** [comment]
That's great to hear!

> The only small thing is that your response for the APOD's uses description instead of explanation

Yeah, that annoys me a bit as well, I used `description` at the beginning of creating this API and then I made an Android app that uses it, but only then I noticed that this field is different in the official API. I wanted to change it, but some users wouldn't be able to load APODs unless they'd update the app, so I left `description` field as is. I don't know if I should change it to `explanation`, I'll think about it (and if I'd change anything, I'd let you know beforehand). I'll surely download your extension, nice thing that it has a version for Firefox
